app/main/forms.py

```diff
@@ -1974,6 +1963,34 @@ class AdminChangeEmailBrandingPoolForm(StripWhitespaceForm):
     )
 
 
+class AdminSetEmailBrandingAddToBrandingPoolStepForm(StripWhitespaceForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        org_name = kwargs['org_name']
+        service_name = kwargs['service_name']
+        branding_name = kwargs['branding_name']
+        self.add_to_pool.param_extensions = {'items': [{'hint': {'html': ''}},
+                                                       {'hint': {'html': ''}}]
+                                             }
+        self.add_to_pool.param_extensions['items'][0]['hint']['html'] = Markup(
+            f'''
+                   Apply branding to {service_name}. <br>
+                   Let other {org_name} teams apply this branding themselves
+                
+                '''
+        )
+        self.add_to_pool.param_extensions['items'][1]['hint']['html'] = Markup(
+            f"Only apply branding to {service_name}")
+
+    add_to_pool = GovukRadiosField(
+        choices=[
+            ('yes', 'Yes'),
+            ('no', 'No')
+        ],
+        validators=[DataRequired(message="Please select an option")]
```

Comment:
Is this the content we normally use for a required selection or otherwise explicitly chosen for this error message?

Reply:
This has been explicitly chosen for this error message. I am agnostic about the content and as @quis noted the content will be looked at and updated later if necessary.

Reply:
You don’t need to explicitly add a validator here because you are using `RadioField`.

When submitting the form `RadioField` will run `pre_validate` which checks that one of the options has been chosen:
https://github.com/alphagov/notifications-admin/blob/601dd09f34e7a4ac54fae89855f2ee17168b1a58/app/main/forms.py#L151-L154

If one hasn’t then it will return the default error message, which is ‘Select an option’

In this case it might be a bit nicer to say ‘Select yes or no’ which you could achieve with the `thing` argument:
 ```suggestion
        thing='yes or no',
```

Here’s one of many examples where we have that content already:
https://github.com/alphagov/notifications-admin/blob/601dd09f34e7a4ac54fae89855f2ee17168b1a58/app/main/forms.py#L1693-L1700

Reply:
Note that ‘Select [thing]’ is the same format used by the GOV.UK Design System: https://design-system.service.gov.uk/components/radios/#error-messages

<img width="770" alt="image" src="https://user-images.githubusercontent.com/355079/190603741-2074ebd5-c965-474f-b985-a7a16442e569.png">

Reply:
Thanks, I will adjust the code accordingly